package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "yarn build:clean && yarn build:tokens && yarn build:js && yarn build:styles && yarn copy-to-lib",
     "build:clean": "rm -rf lib/",
-    "build:tokens": "rm -rf src/tokens-dist/ && node ./style-dictionary.config.js && yarn copy-tokens-to-lib",
+    "build:tokens": "rm -rf src/tokens-dist/ && node ./style-dictionary.config.js && yarn copy-tokens-to-lib && yarn prettier-tokens-dist",
```

Comment:
Nit (really doesn't matter) -

Could run prettier before copying to lib, so that lib and src match.

But it's just formatting differences. Obviously nothing in the logic/values.

Reply:
Makes sense, will move to before copying lib